keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -1341,113 +1454,14 @@ If there is a change in parameters, an event will trigger the following func in
 func (ctrl *resizeController) updateVolumeAttributesClass(vqc *v1.VolumeAttributesClass) (error, bool) {...}
 ```
 
-Under this operation, it will get all the pvcs consuming the volume Performance class and call expandAndRecover to update the volume Performance parameters. The update the volume Performance parameters will overlap with the following path “**Watching changes in the PVC object**”
-
-
-
-
-
-#### CreateVolume
-
-For creating volume, there are two different ways to configure the related parameters. Only **cluster administrators** should be able to create StorageClass and/or VolumeAttributesClass.
-
-
-
-*   From storage class parameters(existing today):
-
-```
-apiVersion: storage.k8s.io/v1
```

Comment:
Do we need to remove the examples form the KEP itself? VAC also has implementation in CreateVolume.

Reply:
We still have this (almost identical text) in the "Provisioned Create" section. I cannot see how this fit in the "Alternative" section.